src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -30,50 +28,61 @@ public void Run(string[] args)
                 ExtendedHelpText = "Welcome to StrykerNet for .Net Core. Run dotnet stryker to kick off a mutation test run"
```

Comment:
Misschien is het handig om in de helptext ergens iets te zeggen over de configfile, dat deze verplicht is en bijv dat er een object in MOET zitten die stryker-config MOET heten binnnen in de toplevel object.

Reply:
Done in the CLIOptions